wezterm-gui/src/main.rs

```diff
@@ -1161,7 +1164,17 @@ fn run() -> anyhow::Result<()> {
     let config = config::configuration();
 
     let sub = match opts.cmd.as_ref().cloned() {
-        Some(sub) => sub,
+        Some(sub) => match sub {
+            // Act as if the normal start subcommand was used,
+            // except that we always start a new instance.
+            // This is needed for compatibility, because many tools assume
+            // that "$TERMINAL -e $COMMAND" blocks until the command finished.
+            SubCommand::BlockingStart(s) => SubCommand::Start(StartCommand {
+                always_new_process: true,
+                ..s
+            }),
+            _ => sub,
+        },
```

Comment:
I think you can simplify this to this:

```suggestion
        Some(SubCommand::BlockingStart(start)) => {
            // Act as if the normal start subcommand was used,
            // except that we always start a new instance.
            // This is needed for compatibility, because many tools assume
            // that "$TERMINAL -e $COMMAND" blocks until the command finished.
            SubCommand::Start(StartCommand {
                always_new_process: true,
                ..start
            })
        },
        Some(sub) => sub,
```